server/consumer.go

```diff
@@ -1891,7 +1891,14 @@ func (o *consumer) deleteNotActive() {
 			interval := consumerNotActiveStartInterval + jitter
 			ticker := time.NewTicker(interval)
 			defer ticker.Stop()
-			for range ticker.C {
+			for {
+				select {
+				case <-ticker.C:
+				case <-o.js.clusterQuitC():
+					return
+				case <-o.srv.quitCh:
```

Comment:
could o.srv or o.js be nil here while attempting the shutdown?

Reply:
As best as I can tell, neither `o.srv` or `o.js` can go back to being `nil`, but I can add extra guards just to be sure.